reg_tests/CMakeLists.txt

```diff
@@ -12,27 +12,41 @@ if(RES_LEN EQUAL 0)
  before running the tests.")
 endif()
 
-set(NALU_GOLD_NORMS_DIR "${CMAKE_CURRENT_SOURCE_DIR}" CACHE PATH
+set(NALU_WIND_REFERENCE_GOLDS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/test_files" CACHE PATH
   "Custom directory for checking gold norms")
 
-if (NOT ${NALU_GOLD_NORMS_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
-  message(STATUS "Setting custom gold norms path: ${NALU_GOLD_NORMS_DIR}")
-  set(TEST_TOLERANCE "1.0e-15")
+if (NOT ${NALU_WIND_REFERENCE_GOLDS_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
This would make sure that if the user specifies a gold norms for, then test tolerances are strict